### ISSUE TITLE
Craft 2 support 

### ISSUE THREAD

**Issue by lucashfreitas** [spicywebau/craft-embedded-assets#162]
Hello,

I have a website using craft 2.9.2 and craft-embedded-assets 0.4.1. Is the plugin supposed to work? I am trying to embed youtube links but all the data is coming empty. 

 I need to update to craft 3 but I was wondering if there is any quick way to update just the plugin to solve the problem as upgrading to craft 3 would take more time to test all the other features.
 
 Thank you very much!

**Comment by ttempleton** [comment]
Yeah, the final release of the Craft 2 version was almost four years ago, so the project dependencies (including the embed-grabbing package that was being used by the plugin at the time) would be fairly out of date at this point, it wouldn't be surprising if some providers no longer worked.  [Development for the Craft 2 version was ended at the release of the Craft 3 version](https://github.com/spicywebau/craft-embedded-assets/pull/57#issuecomment-387370966), before we took over the plugin, so we're not going to provide support for the Craft 2 version ([and also Craft 2 itself is no longer supported outside of critical security issues](https://craftcms.com/knowledge-base/supported-versions)).  However, you are free to fork the project and make any changes you like, including updating the project dependencies and hopefully getting them to a working state.

**Comment by lucashfreitas** [comment]
Thank you very much for the quick reply @ttempleton ! I've forked a version and managed to fix the issues!

**Comment by tlacure17** [comment]
@lucashfreitas I'm having the same issue on some of our sites; any chance I'd be able to see your fixed version? We are in the same boat of not being quite ready to upgrade to Craft 3 due to having to re-evaluate all our sites and change some plugins.

**Comment by lucashfreitas** [comment]
Hey @tlacure17 sorry for taking long to reply. Actually the problem was related to the `embed/embed` plugin. It was out of date, so whenever I tried to load youtube iframe content it was returning an empty/invalid Json. This happens because youtube iframe request must use HTTPS and the old version of the plugin was using HTTP only, so I have replaced the code directly on `vendor` folder and it worked. What is the problem you having atm?
